stan/math/prim/functor/hcubature.hpp

```diff
@@ -270,11 +276,12 @@ inline void make_GenzMalik(const int& dim,
  * @return numeric integral of the integrand, error, and suggested coordinate to
  * subdivide next
  */
-template <typename F, typename T_pars>
+template <typename F, typename ParsTupleT>
 std::tuple<double, double, int> integrate_GenzMalik(
     const F& integrand, std::vector<std::vector<std::vector<double>>>& p,
```

Comment:
I'm seeing this was added in #2838 which was when the larger 7 parameter ddm model was broken up into several smaller PRs so I'm not following why this is out of scope? This is a series of PRs to accomplish one larger task so I think asking about the code for these files in general should be in scope

Reply:
Fair enough, I'll resolve all I previously marked out-of-scope as well. But the key thing is the current structure of the different density & gradient functions, is that alright with you? If so I can add doc and work with @Franzi2114 on updating function/variable names, so that we can get this PR merged for them soon

Reply:
Yes the new structure makes a lot more sense to me. Thank you!